download-references.nf

```diff
@@ -173,14 +189,20 @@ process download_ericscript {
     """
     wget -N https://raw.githubusercontent.com/circulosmeos/gdown.pl/dfd6dc910a38a42d550397bb5c2335be2c4bcf54/gdown.pl
     chmod +x gdown.pl
-    ./gdown.pl "https://drive.google.com/uc?export=download&confirm=qgOc&id=0B9s__vuJPvIiUGt1SnFMZFg4TlE" ericscript_db_homosapiens_ensembl84.tar.bz2
-    tar jxf ericscript_db_homosapiens_ensembl84.tar.bz2
-    rm gdown.pl ericscript_db_homosapiens_ensembl84.tar.bz2
+    if [[ ${params.genome} == "GRCh37" ]]; then
+      ./gdown.pl "https://drive.google.com/uc?export=download&confirm=qgOc&id=0B9s__vuJPvIibDRIb0RFdHFlQmM" ericscript_db_homosapiens_ensembl73.tar.bz2
+      tar jxf ericscript_db_homosapiens_ensembl73.tar.bz2
+    else
+      ./gdown.pl "https://drive.google.com/uc?export=download&confirm=qgOc&id=0B9s__vuJPvIiUGt1SnFMZFg4TlE" ericscript_db_homosapiens_ensembl84.tar.bz2
+      tar jxf ericscript_db_homosapiens_ensembl84.tar.bz2
+    fi
+    rm gdown.pl ericscript_db_homosapiens_ensembl*.tar.bz2
     """
 }
 
 process download_databases {
     publishDir "${params.outdir}/databases", mode: 'copy'
+    executor 'local'
```

Comment:
Should not be here?

Reply:
I had issues as use `PBS/torque`, best probably to remove and I will use in a separate `.conf` for my system.

Reply:
I'll recommend doing a custom profile here: https://github.com/nf-core/configs/